contribs/gmf/src/directives/objectediting.js

```diff
@@ -936,7 +936,8 @@ gmf.ObjecteditingController.prototype.handleSketchFeaturesAdd_ = function(evt) {
     }
 
   } else if (this.process === gmf.ObjecteditingtoolsController.ProcessType.ADD) {
-    this.feature.setGeometry(sketchGeom.clone());
+    this.feature.setGeometry(
+      gmf.ObjecteditingController.toMultiGeometry_(sketchGeom.clone()));
```

Comment:
First, the GMF server throws an error unless we give it a multi geometry.  That's the reason we must ensure that the sent geometry is always multi.

Then, if you check the `toMultiGeometry_`, it already takes care of checking the type of geom and return the same geom if it's already multi, else it creates a multi.

Therefore, I believe the fix is okay.  What do you think?